include/openssl/core_names.h

```diff
@@ -190,6 +190,9 @@ extern "C" {
 #define OSSL_KDF_PARAM_KEY          "key"       /* octet string */
 #define OSSL_KDF_PARAM_SALT         "salt"      /* octet string */
 #define OSSL_KDF_PARAM_PASSWORD     "pass"      /* octet string */
+#define OSSL_KDF_PARAM_PREFIX       "prefix"    /* octet string */
+#define OSSL_KDF_PARAM_LABEL        "label"     /* octet string */
+#define OSSL_KDF_PARAM_DATA         "data"      /* octet string */
```

Comment:
I know this isn't the first time, but it still feels a little weird to use such short/common words as parameters that are specific to the TLS 1.3 HKDF...

Reply:
I half agree.  I'm not too concerned about the strings actually used.  So long as they have some correspondence to what is being passed, I'm okay.

I guess I don't see a problem reusing these labels in other contexts, so long as they are documented each time.